src/steps/measureStep.ts

```diff
@@ -12,16 +12,13 @@ export const measureStep = (
   const stepMetricName = S + step;
   const startMarkExists = WP.getEntriesByName(M + startMark).length > 0;
   const endMarkExists = WP.getEntriesByName(M + endMark).length > 0;
-  if (!endMarkExists || !config.steps) {
+  if (!endMarkExists || !startMarkExists || !config.steps || !config.steps[step]) {
```

Comment:
The reason I ask these questions, is because milliseconds when measuring performance matters, and Perfume should always operate at the lowest coast as possible in CPU.

The last thing we want is that measuring performance is what makes the app slow.